workflow/rules/exposure/network_raster_intersection.smk

```diff
@@ -0,0 +1,50 @@
+"""
+Intersect a network representation with hazard rasters
+"""
+
+
+rule rasterise_osm_network:
+    """
+    Intersect an OSM derived network with a stack of raster maps
+    """
+    conda: "../../../environment.yml"
+    input:
+        network="{OUTPUT_DIR}/geoparquet/{DATASET}_{FILTER_SLUG}/processed/{SLICE_SLUG}_edges.geoparquet",
+        tif_paths=rules.trim_hazard_data.input.trimmed_rasters,
+    params:
+        copy_raster_values=True
+    output:
+        geoparquet="{OUTPUT_DIR}/splits/{DATASET}_{FILTER_SLUG}/{HAZARD_SLUG}/{SLICE_SLUG}.geoparquet",
+        parquet="{OUTPUT_DIR}/splits/{DATASET}_{FILTER_SLUG}/{HAZARD_SLUG}/{SLICE_SLUG}.parquet",
+    # intersection.py is intermittently failing, use retries as temporary fix
+    # TODO: investigate why this job is sometimes failing with a coredump from intersection.py
```

Comment:
Could you create an issue for this? Do we have an example input that will reliably fail (even some of the time), or coredumps from past failures?

Reply:
I've just had a look through my notebook and unfortunately I didn't write down which input was intermittently breaking it. Will make an issue.